src/Umbraco.Tests.Common/Builders/UserGroupBuilder.cs

```diff
@@ -92,7 +92,7 @@ public IReadOnlyUserGroup BuildReadOnly(IUserGroup userGroup)
 
         public override IUserGroup Build()
         {
-            var id = _id ?? 1;
+            var id = _id ?? 0;
```

Comment:
0 is intended.

This is the userGroup builder... But for user builder 0 is also intended, so it can be created by default. If you need an admin, then you can use WithId(Constants.Security.SuperUserId)